share/spice/skyscanner_flight_search/skyscanner_flight_search.css

```diff
@@ -0,0 +1,38 @@
+/*
+    Document   : zeroclickinfo-spice/share/spice/skyscanner_flight_search/skyscanner_flight_search.css
+    Created on : 2016-09-23 11:50 AM
+    Author     : sheyman
+    Description:
+    Purpose of the stylesheet follows.
+    To change this template use Tools | Templates.
+*/
+
+.tile_title {
+}
+
+.tile__title__sub {
+    color:#999 !important;
+}
+
+.tile__foot {
+    max-height: 4.37em !important;
+}
+
+@media only screen and (max-width: 590px) {
+    .tile__media__img {
+        max-width: 139% !important;
+    }
+}
+
+.price {
+    color: #333;    
```

Comment:
We have existing CSS classes for certain colors. We need to use our classes so that the colours adjust correctly for the other DDG UI themes (like dark mode). The classes can be applied in the Handlebars

The list of classes can be seen here: https://duckduckgo.com/styleguide

You'll want `tx-clr--slate`